canonical-web-and-design/demos.haus
# Switch to dart sass

## Done

Switch to dart sass

## QA

- Check out this feature branch
- `docker build -t demos-haus .` and see the image builds successfully
